UPGRADE.md

```diff
@@ -1,3 +1,10 @@
+### UPGRADE FROM 0.24.X TO 1.0.0
```

Comment:
We will probably release support for Sylius 2.0 in version 2.0 of this plugin, and release earlier 1.0, but we could adjust it later 🖖🏻